File: _java/java_basic/src/main/java/ch00/SwitchValueExample.java
package ch00;

import java.util.Scanner;

class SwitchValueExample {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String grade = scanner.nextLine();

        //Java 11 이전 문법
        int score1 = 0;
        switch (grade) {
            case "A" -> score1 = 100;
            case "B" -> score1 = 100 - 20;
            default -> score1 = 60;
        }

        System.out.println("score1: " + score1);
    }


}

File: _java/java_basic/src/main/kotlin/SwitchValueExample.kt
fun main(){
    val grade = "B"

//Java 11 이전 문법
    var score1 = 0
    when (grade) {
        "A" -> score1 = 100
        "B" -> {
            val result = 100 - 20
            score1 = result
        }
        else -> score1 = 60

    }
    println("score1: ${score1}")
}